daemon/api.go

```diff
@@ -327,6 +332,14 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 		}
 	}
 
+	if len(includeTypes) > 0 {
```

Comment:
it's probably

https://search.apps.ubuntu.com/api/v1/search?q=content:%22framework%22%20content:%22gadget%22


On 25 January 2016 at 14:41, Steven Wilkin <notifications@github.com> wrote:

> In daemon/api.go
> <https://github.com/ubuntu-core/snappy/pull/366#discussion_r50700545>:
>
> > @@ -327,6 +332,14 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
> >  		}
> >  	}
> >
> > +	if len(includeTypes) > 0 {
>
> Having a space between multiple search terms seems to work:
>
> $ curl -s 'https://search.apps.ubuntu.com/api/v1/search?q=content:framework%20gadget' | jq '._embedded["clickindex:package"][].content' | uniq
> "gadget"
> "framework"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubuntu-core/snappy/pull/366/files#r50700545>.
>